client/components/PubShareDialog/PubShareDialog.tsx

```diff
@@ -68,14 +71,32 @@ const AccessHashOptions = (props: AccessHashOptionsProps) => {
 		);
 	};
 
+	const pubUrlFunction = (hash) =>
+		pubUrl(communityData, pubData, {
+			accessHash: hash,
+			isDraft: !isRelease,
+		});
+	const reviewUrlFunction = (hash) =>
+		pubUrl(communityData, pubData, {
+			accessHash: hash,
+			historyKey: historyData.currentKey,
+			isReview: true,
+		});
+
+	const renderRow = (label, hash, urlFunction) => {
+		const url = urlFunction(hash);
+		return renderCopyLabelComponent(label, url);
+	};
```

Comment:
We may be able to remove the `*UrlFunction` functions and the `renderRow` function by sticking to plain old variables. IMO it makes the code a bit easier to follow! ie.

```ts
const isDraft = !isRelease
const createAccessUrl = (accessHash, options) => pubUrl(communityData, pubData, { accessHash, ...options })
const reviewAccessUrl = getAccessUrl(viewHash, { historyKey: historyData.currentKey, isReview: true })
// ...
{viewHash && renderCopyLabelComponent('View', createAccessUrl(viewHash, { isDraft }))}
{editHash && renderCopyLabelComponent('Edit', createAccessUrl(editHash, { isDraft }))}
{renderCopyLabelComponent('Review', reviewAccessUrl)}
```